pages/editor.vue

```diff
@@ -19,7 +17,7 @@
     const comicStore = useComicStore();
     const catalogElements = ref([]);
     const catalogStructure = ref([]);
-    const comic = reactive(comicStore.comic);
+    const comic = reactive(toRaw(comicStore.comic));
```

Comment:
I am not sure what the purpose is of first setting it to not reactive and ,make it reactive afterwards.